### PR TITLE
refactor: replaced SKL with sFuel as a gas token in logs

### PR DESCRIPTION
**Pull request by Yashk767**
# Description

Currently SKL is being logged as  gas token, SKL is never used as an gas ans sfuel and the SKL token are different.
Replaced SKL with sFuel in logs.

### CODE REVIEW

**Comment by yohanelly95** [comment]
@Yashk767 @ashish10677 the min amount will most probably be 0.1 sfuel since the faucet disperses only 0.1 sfuel. Change it to 0.001 sfuel or so imo

**Comment by Yashk767** [comment]
> @Yashk767 @ashish10677 the min amount will most probably be 0.1 sfuel since the faucet disperses only 0.1 sfuel. Change it to 0.001 sfuel or so imo

True, makes sense

**Review by ashish10677 — APPROVED** [review]
lgtm

**Comment by rajkharvar** [comment]
@Yashk767 There are other instances where 'sFUEL' is being used. Could you perform a global search and update them as well?

**Comment by ashish10677** [comment]
> @Yashk767 There are other instances where 'sFUEL' is being used. Could you perform a global search and update them as well?

Using sFuel is fine. Using SKL isn't.

**Review by rajkharvar — DISMISSED** [review]

**Review by yohanelly95 — APPROVED** [review]

File: cmd/vote.go [hunk 1]
```diff
@@ -215,9 +215,9 @@ func (*UtilsStruct) HandleBlock(client *ethclient.Client, account types.Account,
 		return
 	}
 
-	// Warning the staker if ETH balance is less than 0.1 ETH
-	if ethBalance.Cmp(big.NewInt(1e17)) == -1 {
-		log.Warn("sFuel balance is lower than 0.1 SKL, kindly add more SKL to be safe for executing transactions successfully")
+	// Warning the staker if sFuel balance is less than 0.001 sFuel
+	if ethBalance.Cmp(big.NewInt(1e15)) == -1 {
```
Comment (yohanelly95) [inline cmd/vote.go:219]: @Yashk767 should be `sFUEL` this is from Skales portal

**Review by yohanelly95 — CHANGES_REQUESTED** [review]

File: cmd/vote.go (hunk 1 above)
Comment (Yashk767) [reply cmd/vote.go:219]: I think its fine as its a variable name `ethBalance` as on oracle node all the variables and functions are related to gas are named with eth  like `var ethBalance`, `func CheckEthBalanceIsZero()`. This should work right?

While in logs and comments we have changed it to sFuel wherever required.

Comment (Yashk767) [reply cmd/vote.go:219]: @yohanelly95 does this make sense? elsewhere SKL is changed to sFuel wherever needed in comments and logs.

Comment (Yashk767) [reply cmd/vote.go:219]: ohh did you mean to use `sFUEL` instead of `sFuel` ?

Comment (yohanelly95) [reply cmd/vote.go:219]: yes use sFUEL instead of sFuel @Yashk767

Comment (Yashk767) [reply cmd/vote.go:235]: Done in https://github.com/razor-network/oracle-node/pull/1253/commits/93487e9adf49659f95cf6d37a26813618863a8a1

**Review by yohanelly95 — APPROVED** [review]

**Review by ashish10677 — APPROVED** [review]

**Review by rajkharvar — APPROVED** [review]

**Pull request merged**